tincr/io/library/genCellLibrary.tcl

```diff
@@ -334,7 +438,26 @@ proc createCellLibrary { {part xc7a100t-csg324-3} {filename ""} } {
         puts "Processing: $cname"
 
         puts $fo "    <cell>"
-        puts $fo "      <type>[get_property NAME $libcell]</type>"
+        set cname [get_property NAME $libcell]
+        puts $fo "      <type>$cname</type>"
+
+        # Mark LUT cells
+        if { [string first "LUT" $cname] == 0 } {
```

Comment:
I would change this to: 
`if { [get_property PRIMITIVE_GROUP $libcell] == "LUT" }` 
I think it's better to use the vivado properties instead of string parsing (which is not guaranteed to always work)